explorer/migrations/0009_auto_20220830_1545.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('explorer', '0008_auto_20190308_1642'),
```

Comment:
```suggestion
        ('explorer', '__latest__'),
```

This migration is the difficult part in porting this to the main branch because it adds a migration in to the past. 

My current thinking is that by depending on the latest migration, this commit will work on this 1.x release or on the current release.

There is a chance you'll not be able to upgrade from this new version, but that might be acceptable to you & we could still release changes on this branched code.

Reply:
> My current thinking is that by depending on the latest migration, this commit will work on this 1.x release or on the current release.

Good point on the migrations, that was something I had not considered. I will need to confirm with my manager, but Im thinking that we will want these changes to be a part of the current release and not `1.x` release due to the migration challenges as we will be upgrading from Django 1.11 to Django 2.2 in the near future and will also want to upgrade `django-sql-explorer` at that time. I suppose I can import the forked commit into our project and then after we upgrade to Django 2.2, I can then upgrade to the current release. I will of course have to migrate the data at that time, but that is pretty straightforward to do.

All that being said, I will get back to you to confirm that we want to go forward on the current release.

Reply:
@marksweb I talked to my manager and he thinks it best to make this a part of the current release and not `1.x`. That being said, I looked at the [current code](https://github.com/groveco/django-sql-explorer) and I dont think this code will auto merge into that due to the [views being separated](https://github.com/groveco/django-sql-explorer/tree/master/explorer/views) instead of in a single `views.py` file like it is in `1.1.3` release. 

So Im thinking it best to do the changes based off the master branch and then submit a new PR. What are your thoughts on that?

Reply:
Ok this makes sense, branch from master and you can use this PR as reference to bring the code into that branch & then create the migration again once things are in place 👍